samples/helloworld/enclave/CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# TODO: Where is the best place to set this
```

Comment:
Set what?

Reply:
My bad. I deleted the line of code after it but did not delete the comment.